### PR TITLE
Add python3 compatibility variable in state.py and update network.py

### PR DESCRIPTION
**Pull request by osamacis**
Add python3 compatibility variable in `state.py `and update `network.py`

### CODE REVIEW

File: src/bitmessagekivy/baseclass/network.py [hunk 1]
```diff
@@ -17,7 +17,8 @@
     from pybitmessage.mock import kivy_main
     stats = kivy_main.network.stats
     objectracker = kivy_main.network.objectracker
-else:
+
+if state.py3_compatible:
```
Comment (PeterSurda) [inline src/bitmessagekivy/baseclass/network.py:21]: maybe `state.backend_py3_compatible`

**Review by PeterSurda — CHANGES_REQUESTED** [review]

File: src/bitmessagekivy/baseclass/network.py (hunk 1 above)
Comment (osamacis) [reply src/bitmessagekivy/baseclass/network.py]: yes now i have forcefully pushed the changed and now it is same as suggested by you.

**Review by PeterSurda — CHANGES_REQUESTED** [review]
Ok but one thing is missing, it needs to be set `True` when running kivy tests.

File: src/bitmessagekivy/baseclass/network.py [hunk 2]
```diff
@@ -13,12 +13,10 @@
 
 from pybitmessage import state
 
-if os.environ.get('INSTALL_TESTS', False):
+if os.environ.get('INSTALL_TESTS', False) and not state.backend_py3_compatible:
     from pybitmessage.mock import kivy_main
     stats = kivy_main.network.stats
     objectracker = kivy_main.network.objectracker
-else:
```
Comment (PeterSurda) [inline src/bitmessagekivy/baseclass/network.py:20]: this section should remain

**Review by PeterSurda — CHANGES_REQUESTED** [review]

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**